xml/Microsoft.Vsa/BaseVsaEngine.xml

```diff
@@ -30,7 +30,7 @@
     </Attribute>
```

Comment:
one idea for this class
https://docs.microsoft.com/en-us/dotnet/api/microsoft.jscript.vsa.basevsaengine?view=netframework-4.8 seems to have a lot of the same APIs. I think we should be able to use the same descriptions.